### ISSUE TITLE
Find in file input text invisible [insiders]

### ISSUE THREAD

**Issue by Eldair** [microsoft/vscode#126896]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.58
- OS Version: Win 10 19043.1081

Steps to Reproduce:

1. Find in file input text is invisible when focused

![image](https://user-images.githubusercontent.com/470003/122913016-3fbf8c00-d359-11eb-97ab-c48eb43bab8b.png)
![image](https://user-images.githubusercontent.com/470003/122913063-4bab4e00-d359-11eb-8a93-c25b76bb33df.png)
![image](https://user-images.githubusercontent.com/470003/122913078-50700200-d359-11eb-921a-bfe7fe81bd22.png)

**Comment by rebornix** [comment]
https://github.com/microsoft/vscode/blob/89d620f4fc6c13599e95dbbde1a044c3c9d1139b/src/vs/workbench/contrib/terminal/browser/media/terminal.css#L27-L27 caused this.

**Comment by alexdima** [comment]
:+1: I also bisected to this one:
```
9dc76ccce1de1a72473f1ba2a38e0c041939e1e8 is the first bad commit
commit 9dc76ccce1de1a72473f1ba2a38e0c041939e1e8
Author: Daniel Imms <2193314+Tyriar@users.noreply.github.com>
Date:   Mon Jun 21 06:41:31 2021 -0700

    Hide terminal editor textarea

    Fixes #126701

 src/vs/workbench/contrib/terminal/browser/media/terminal.css | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)
```

**Comment by charkour** [comment]
Thanks for the quick fix!
